### PR TITLE
Feature/crud jobseeker

### LINKED ISSUE #6: Create CRUD for job seeker
similar as [#7](https://github.com/PAWII-Portal-Loker/backend/issues/7),
start to work on this after that issue done

### LINKED ISSUE #19: Add last education enums

### PR DESCRIPTION
**Pull request by oohgan**
Please acc

### CODE REVIEW

File: src/modules/jobSeeker/services/jobSeeker.service.ts [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+    if (!updatedUser) {
+      this.delete({ _id: newJobSeeker.id });
+      return this.throwError("Error updating user", StatusBadRequest)
+    }
+
+    return newJobSeeker._id as string;
+  }
+
+  public async updateJobSeeker(
+    data: Partial<JobSeekerCreateDto>,
```
Comment (fanesz) [inline src/modules/jobSeeker/services/jobSeeker.service.ts]: should be using JobSeekerUpdateDto

File: src/modules/jobSeeker/services/jobSeeker.service.ts [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+    const updatedJobSeeker = await this.update(
+      { _id: jobSeeker.id },
+      {
+        name: data.name,
+        lastEducation: data.lastEducation,
+        major: data.major,
+        gpa: data.gpa,
+      },
+    );
+    console.log(updatedJobSeeker);
```
Comment (fanesz) [inline src/modules/jobSeeker/services/jobSeeker.service.ts]: remove cl

**Review by fanesz — CHANGES_REQUESTED** [review]
no controller and dto for update yet.
make sure to test it on postman

File: src/modules/jobSeeker/services/jobSeeker.service.ts (hunk 2 above)
Comment (oohgan) [reply src/modules/jobSeeker/services/jobSeeker.service.ts]: image
image
image

done

**Pull request merged**